qemu/tests/qemu_guest_agent.py

```diff
@@ -1918,8 +1946,129 @@ def gagent_check_fsfreeze(self, test, params, env):
                 logging.warn("Finally failed to thaw guest fs,"
                              " detail: '%s'", detail)
             raise
+        # check after fsthaw
+        self._action_after_fsthaw(write_cmd_guest, write_cmd_timeout)
+
+    @error_context.context_aware
+    def gagent_check_fsfreeze(self, test, params, env):
+        """
```

Comment:
@xiagao  you have added command guest-fsfreeze-freeze-list  to avocado-vt. why no use it directly ?

Reply:
@xiagao move this comment to 1974.